venue.html.j2

```diff
@@ -15,6 +15,25 @@
     </small>
   </figure>
 
+  <h2>Hotels</h2>
+
+  The conference has not reserved a hotel block for this year but has negotiated rates with a few local hotels.
+  If you are using the hotels listed below or finding your own accomodation we recommend that you book early.
+
+  <h3>Alt Hotel Montreal</h3>
+  <p>
+  <a href="https://www.germainhotels.com/en/le-germain-hotel/montreal">The Alt Hotel</a> is located
+  at 120, rue Peel, approximately a 15 minute walk from the conference venue.  Rooms are available
+  starting at $239 CAD a night.  You can book with this <a href="https://reservation.germainhotels.com/ibe/details.aspx?propertyid=17532&nights=1&checkin=05/12/2025&group=2505SQLDEV&lang=en-us&adults=1&childAges=">link</a>
+  Bookings with the above link must be made by April 12 2025.  It might be possible to make bookings by phone or email using the code 2505SQLDEV after this date
```

Comment:
```suggestion
  Bookings with the above link must be made by April 12, 2025.  It might be possible to make bookings by phone or email using the code `2505SQLDEV` after this date
```